src/Semantic/Git.hs

```diff
@@ -17,62 +17,74 @@ module Semantic.Git
 
 import Prologue
 
-import           Data.Attoparsec.Text (Parser)
-import           Data.Attoparsec.Text as AP
+import           Data.Attoparsec.ByteString (Parser)
+import           Data.Attoparsec.ByteString as AP
+import           Data.ByteString (ByteString)
+import           Data.ByteString.Internal (w2c)
```

Comment:
`w2c` is a no-op. (`c2w` is its silently-truncating cousin; this module doesn’t use that).